### PR TITLE
Improve zh-tw (Traditional Chinese) locale

### PR DESCRIPTION
**Pull request by PeterDaveHello**
Improve zh-tw (Traditional Chinese) locale

Please indicate you've done the following:
- [x] Well Written Title and Summary of the PR
- [x] Label the PR as needed. "release-note/ignore-for-release, release-note/new-feature, release-note/update, release-note/enhancement, release-note/community, release-note/breaking-change, release-note/docs, release-note/infra, release-note/deprecation"
- [x] Accepted the DCO. Commits without the DCO will delay acceptance.
- [x] Made sure tests are passing and test coverage is added if needed.
- [ ] Considered the docs impact and opened a new docs issue or PR with docs changes if needed in [website repository](https://github.com/goharbor/website).

### CODE REVIEW

**Comment by pichuang** [comment]
As a long-time user of the zh-tw language, I personally recommend this PR.

**Comment by AllForNothing** [comment]
@PeterDaveHello Thanks for your contribution.
Handling of keys and parameters looks ok.  Can you invite someone to help to review your translation?

**Comment by wy65701436** [comment]
Thank you, @pichuang, for your comments. Please go ahead and add an LGTM. Additionally, we would appreciate another community member who uses the zh-tw language to review it. By the way, I am generally satisfied with these changes.

**Review by wy65701436 — APPROVED** [review]
thanks, LGTM

**Review by chlins — APPROVED** [review]
lgtm

File: src/portal/src/i18n/lang/zh-tw-lang.json [hunk 1]
```diff
@@ -1420,427 +1419,427 @@
...
+    "SCHEDULE_PAUSE_BTN_INFO": "暫停 - 暫停所有排程的執行。",
+    "SCHEDULE_RESUME_BTN_INFO": "繼續 - 恢復所有排程的執行。",
+    "WORKER_FREE_BTN_INFO": "停止目前執行的工作以釋放 worker",
     "CRON": "Cron",
-    "WAITING_TOO_LONG_1": "Certain jobs have been pending for execution for over 24 hours. Please check the job service ",
-    "WAITING_TOO_LONG_2": "dashboard.",
-    "WAITING_TOO_LONG_3": "For more details, please refer to the ",
-    "WAITING_TOO_LONG_4": "Wiki."
+    "WAITING_TOO_LONG_1": "某些工作等待執行超過 24 小時。請檢查工作服務儀表板。",
+    "WAITING_TOO_LONG_2": "更多詳細資訊，請參考 Wiki。",
```
Comment (AllForNothing) [inline src/portal/src/i18n/lang/zh-tw-lang.json:1791]: "dashboard"  and "Wiki" are hyperlink text， so they should be stand-alone items

File: src/portal/src/i18n/lang/zh-tw-lang.json [hunk 2]
```diff
@@ -1420,427 +1419,427 @@
...
+    "CRITERIA": "標準",
+    "ONLY_SIGNED": "僅限已簽名的映像檔",
+    "PREHEAT_ON_PUSH": "推送時預熱",
+    "START_TEXT": "無漏洞嚴重性",
+    "EDN_TEXT": "及以上",
+    "LABELS": "標籤",
+    "SCHEDULE": "排程",
+    "TEST_FAILED": "測試失敗",
+    "MANUAL": "手動",
+    "SCHEDULED": "已排程",
```
Comment (AllForNothing) [inline src/portal/src/i18n/lang/zh-tw-lang.json:1560]: This item is used to describe the task type.
image

Comment (PeterDaveHello) [reply src/portal/src/i18n/lang/zh-tw-lang.json:1560]: Updated, thanks.

File: src/portal/src/i18n/lang/zh-tw-lang.json (hunk 1 above)
Comment (PeterDaveHello) [reply src/portal/src/i18n/lang/zh-tw-lang.json]: Updated, thanks.

**Comment by PeterDaveHello** [comment]
Hey guys, I sent a follow up of this PR - #19161 to continue the improvement works of zh-tw locale here, would be great to have your reviews, thanks.

**Pull request merged**